### ISSUE TITLE
Custom descriptions/labels NVDA

### ISSUE THREAD

**Issue by peterpacult** [nvaccess/nvda#12488]
In JAWS you can create custom descriptions for graphics, form fields or text links (https://www.freedomscientific.com/SurfsUp/Custom_Labels.htm). It would be very helpful if NVDA also had this functionality. Either directly in the programme or possibly with the help of an add-on.

**Comment by cary-rowen** [comment]
There are many software developed that do not follow the accessibility standards and may not add accurate leabel to the button. If you can use this function to add it yourself, it may be good, including google Talkback also has such a function.

**Comment by XLTechie** [comment]
As does iOS Voiceover, though it doesn't work some of the time.

**Comment by lijianwei2019** [comment]
repeat #2111

**Comment by Adriani90** [comment]
closing as duplicate of #2111
